### PR TITLE
PREAPPS-2733 Moving client from build time to run time

### PR DESCRIPTION
**Pull request by itsmeprashant**
Refer: https://jira.corp.synacor.com/browse/PREAPPS-3555

### CODE REVIEW

File: css-variables.less [hunk 1]
```diff
@@ -1,192 +1,126 @@
...
-	// Large screen / wide desktop
-	--screen-lg: @screen-lg;
-	--screen-lg-desktop: @screen-lg-desktop;
-	// So media queries don't overlap when required, provide a maximum
-	--screen-xs-max: @screen-xs-max;
-	--screen-sm-max: @screen-sm-max;
-	--screen-md-max: @screen-md-max;
-}
\ No newline at end of file
+:root{
```
Comment (silentsakky) [inline css-variables.less]: we should remove generate-css-variables.js or make it inverse

File: variables.less [hunk 2]
```diff
@@ -1,189 +1,201 @@
...
+@logo-max-width: var(--logo-max-width);
 
 // Spacing
-@spacing-xs: 4px;
-@spacing-sm: 8px;
-@spacing-md: 16px;
-@spacing-lg: 24px;
-@spacing-xl: 32px;
-
+@spacing-xs:4px;
```
Comment (silentsakky) [inline variables.less]: var ?

Comment (itsmeprashant) [reply variables.less]: not possible .. cz these variables are written in media query and which doesn't support var

here- https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries

File: css-variables.less (hunk 1 above)
Comment (itsmeprashant) [reply css-variables.less]: yes, eventually it will go away from here.

**Pull request merged**